userspace/libsinsp/test/sinsp_with_test_input.h

```diff
@@ -272,7 +272,7 @@ class sinsp_with_test_input : public ::testing::Test {
 		return chk->extract(evt, values);
 	}
 
-	std::string get_field_as_string(sinsp_evt *evt, const std::string& field_name)
+	std::string get_field_as_string(sinsp_evt *evt, const std::string& field_name, bool return_null_result = false)
```

Comment:
Why is this necessary? Could we use `field_exists` instead or does that one have some issues? I don't think it's wrong but I think it would be better not to get case of `"null"` vs non existent in tests as well :sweat_smile:

Reply:
oops, no that was just me not looking around in the tests for best option, fixed it, thanks @LucaGuerra!